skyuxconfig.json

```diff
@@ -14,8 +14,6 @@
         "text": "Suggest edit",
         "url": "https://github.com/blackbaud/skyux2-docs"
       },
-      "footer": {
```

Comment:
Why are we deleting the footer? We just added it to address https://github.com/blackbaud/skyux2/issues/2117. The issue doesn't have any details on where the request to add the footer came from, but I remember someone requested it. It might have been Lindsey. Should I look into this? Or is there a strong reason why we don't want a footer on this site?

Reply:
Agreed.  I got too aggressive and just needed to remove a double instance from the homepage.  It's worth noting though that the footer does not currently show up on our site as we're not using a high enough version of Stache.

Reply:
The footer showed up when I served locally; I didn't notice it wasn't showing up on the live site. This PR updates the Stache version, and when I serve locally, the footer is in place. So I assume this PR will fix the missing footer on the live site.